### PR TITLE
Tweedie tutorial

### PR DESCRIPTION
**Pull request by ilya-pchelintsev**
I hereby agree to the terms of the CLA available at: [https://yandex.ru/legal/cla/?lang=en](https://yandex.ru/legal/cla/?lang=en).

### CODE REVIEW

File: catboost/tutorials/regression/tweedie.ipynb [hunk 1]
```diff
@@ -0,0 +1,398 @@
+{
```
Comment (LyzhinIvan) [inline catboost/tutorials/regression/tweedie.ipynb]: <p>calim -&gt; claim</p><p>out targer -&gt; our target</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catboost/catboost/pull/1232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by LyzhinIvan** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
CLA already signed

**Comment by arcadia-devtools** [comment]
@LyzhinIvan, internal review request created: [1191379](https://a.yandex-team.ru/review/1191379)

**Comment by arcadia-devtools** [comment]
Check [1191379](https://a.yandex-team.ru/review/1191379) passed

**Comment by LyzhinIvan** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@LyzhinIvan, internal review request created: [1192018](https://a.yandex-team.ru/review/1192018)

**Comment by arcadia-devtools** [comment]
Check [1192018](https://a.yandex-team.ru/review/1192018) passed

**Pull request merged**